src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.LVA.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        public enum LVA : uint
+        {
+            DEFAULT = 0x0000,
+            ALIGNLEFT = 0x0001,
```

Comment:
Can we use the following to align with the [docs](https://docs.microsoft.com/en-us/windows/win32/controls/lvm-arrange#parameters)?
```suggestion
            ALIGNLEFT = (uint)LVS.ALIGNLEFT,
```

Reply:
Note sure this is right:
```
#define LVS_ALIGNTOP            0x0000
#define LVS_ALIGNLEFT           0x0800
#define LVS_ALIGNMASK           0x0c00
```

In our code:
```
        LVS_ALIGNTOP = 0x0000,
        LVS_ALIGNLEFT = 0x0800,
```

Reply:
Scratch this, I misread the docs. All good.